src/paper/serializers/paper_serializers.py

```diff
@@ -483,7 +482,14 @@ def create(self, validated_data):
                         filters=[NEWEST],
                         with_default_hub=True,
                     )
-
+                unified_doc = ResearchhubUnifiedDocument.objects.create(
+                    document_type="paper.Paper",
+                    hot_score=paper.calculate_hot_score(),
+                    score=paper.score,
+                )
+                unified_doc.hubs.add(*hub_ids)
```

Comment:
```suggestion
                unified_doc = ResearchhubUnifiedDocument.objects.create(
                    document_type="paper.Paper",
                    score=paper.score,
                )
                unified_doc.calculate_hot_score_v2(should_save=True)
                unified_doc.hubs.add(*hub_ids)
```

Reply:
It should default to 0 if it's not included?

Reply:
No, @yattias mentioned that there's some sort of sync method between v2 and the old one. I dont want to mess with it just now. If there needs to be a migration, it can be part of a separate branch